### PR TITLE
Make the Nrdb Timeout Configurable

### PR DESCRIPTION
**Pull request by xqi-nr**
# Description
Some customer observed the error `"NRDB query duration exceeded the set timeout"` in the metrics-adapter log. 

# Code Changes
The NRDB client's timeout is configurable. This PR expose the NRDB client's timeout through a chart config parameter.

# Tests
Test it locally with tilt
```
make tilt-up 
```
Add logs to print out the nrdb timeout value to make sure the config parameter from the values.yaml take effects

image

### CODE REVIEW

File: charts/newrelic-k8s-metrics-adapter/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
-version: 1.1.2
-appVersion: 0.3.0
```
Comment (htroisi) [inline charts/newrelic-k8s-metrics-adapter/Chart.yaml:5]: Can you revert this app & chart version bump and we'll just pull this into the next release?

**Review by htroisi — CHANGES_REQUESTED** [review]
@xqi - For this PR, can you do the following: 
1. Revert the changes to the `Chart.yaml` file in this PR. Merging a PR with a helm chart `version` bump will automatically release a new helm chart. Before we do that, we need to  manually release the app. 
2. Merge the app & chart changes.
3. Manually release the app (following the directions in the `K8s Agents Release` doc).
4. Release the chart (following the directions in the `K8s Agents Release` doc).

**Comment by xqi-nr** [comment]
> @xqi - For this PR, can you do the following:
> 
> 1. Revert the changes to the `Chart.yaml` file in this PR. Merging a PR with a helm chart `version` bump will automatically release a new helm chart. Before we do that, we need to  manually release the app.
> 2. Merge the app & chart changes.
> 3. Manually release the app (following the directions in the `K8s Agents Release` doc).
> 4. Release the chart (following the directions in the `K8s Agents Release` doc).

Thanks for the comments. I revert the chart version change and only keep the app version change.

File: charts/newrelic-k8s-metrics-adapter/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
 version: 1.2.0
-appVersion: 0.4.2
```
Comment (htroisi) [inline charts/newrelic-k8s-metrics-adapter/Chart.yaml:5]: Revert this appVersion bump. We do a single appVersion bump during the release process.

**Review by htroisi — APPROVED** [review]

File: charts/newrelic-k8s-metrics-adapter/ci/test-values.yaml [hunk 3]
```diff
@@ -7,6 +7,7 @@ verboseLog: false
 config:
   accountID: 111
   region: EU
+  nrdbClientTimeoutSeconds: 60
```
Comment (htroisi) [inline charts/newrelic-k8s-metrics-adapter/ci/test-values.yaml:10]: any reason we don't use our default value for testing?

Comment (xqi-nr) [reply charts/newrelic-k8s-metrics-adapter/ci/test-values.yaml:10]: no, let me update it to 30

**Review by juanjjaramillo — APPROVED** [review]

**Pull request merged**